yax/core/src/main/scala/doobie/hi/resultset.scala

```diff
@@ -206,7 +206,7 @@ object resultset {
       val b = Vector.newBuilder[A]
       while (n > 0 && rs.next) {
```

Comment:
BTW, it'll yield empty vector when a chunkSize <= 0 is given. Should it be prevented at all or be handled as a special case?